done 0.1

Fugas252mm opened the pull request.

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,295 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="true">
```
Need to fix it everywhere
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="true"
    >
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,295 @@
+  <!-- #region header -->
```
Comment is redundant here, so you need to remove it everywhere
```suggestion
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,295 @@
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
```
Need to fix format phone everywhere only in href
```suggestion
          <a href="tel:+123455555555" class="menu__phone-number">
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,295 @@
+      </main>
+    </div>
+
+    <footer class="footer">
+      <img
+        src="../src/images/footer.png.png"
+        alt="footer"
+        class="footer__img"
+      >
+    </footer>
```
Add image as a background for the footer
```suggestion
    <footer class="footer">
    </footer>
```

volodymyr-soltys97 reviewed (changes requested):
Add the demo link to the PR description and to the commit
<img width="962" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/64750958/99c197dc-a52c-45a8-b9cf-b1dcb1733303">

anastasiiavorobiova reviewed (changes requested):
Great start! Keep doing: 
<img width="1072" alt="Screenshot 2023-12-26 at 16 21 23" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/5c0cebc4-ac39-412f-a060-02753ff2dced">

<img width="1484" alt="Screenshot 2023-12-26 at 16 21 29" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/dd79af92-23b9-438e-bf32-1021929b21c5">
Consider fixing validation errors 
<img width="901" alt="Screenshot 2023-12-26 at 16 20 04" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/f8d64e89-bee6-44e4-9d0b-8029c813c9ed">
Check the background position for tablets 
<img width="827" alt="Screenshot 2023-12-26 at 16 19 37" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/dc6b10b0-c71b-48f9-8c37-f05fe8cab3e1">
A user should be able to scroll the menu in landscape mode 
<img width="292" alt="Screenshot 2023-12-26 at 16 19 23" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/021fd34a-6036-4f78-8ef9-0d58f6698a2a">
Consider adding hovers here 
<img width="786" alt="Screenshot 2023-12-26 at 16 18 29" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/c6de266f-454f-4181-adf5-47479977f47b">
Consider removing default outlines and default borders here. Check font for message field. Please, add the required attribute for form fields 
<img width="511" alt="Screenshot 2023-12-26 at 16 18 12" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/bf88f6ba-712a-4b29-9552-3eb4db3ee169">
Consider adding proper href attribute here 
<img width="301" alt="Screenshot 2023-12-26 at 16 18 03" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/2c12c6b8-578d-4e65-86fc-9cb2e8f7660f">
Consider exporting an image without a label 
<img width="629" alt="Screenshot 2023-12-26 at 16 17 59" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/24adde5b-0de1-4c9d-9621-e688b8402da1">
Check the font family for your page 
<img width="199" alt="Screenshot 2023-12-26 at 16 17 48" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/adf91ab2-8d17-4cc7-87bb-52ee57ad06c1">
Add hovers here

Fugas252mm commented:
it is not possible to upload a picture without the icon at the bottom

sTorba24 reviewed (changes requested):
1.export it as jpg
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/d994a846-a8ab-41e0-980e-b1e20eac9c61)


2. click on phone should trigger call
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/d178b428-d755-4427-9376-8b53a7c69669)


3.add hover effects on buttons and cursor pointer to all interactive elements

4.  it should be links
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/299a1780-26f1-45b6-b506-f4bfef936499)


5. add max-width for content, it shouldn't be stretched for full width on huge screens
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/042c0ffa-97d2-441b-9280-11df7b3b94d8)


6. form should be submit after click on button and clear then

Fugas252mm commented:
![image](https://github.com/mate-academy/layout_landing-page/assets/143828583/995259cc-0a30-4a4f-92f7-d3e93ee42359)
the picture with the bottom element is loaded anyway

l4st1m0za reviewed (changes requested):
1) Add right title of the page here
<img width="241" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/844fa133-f517-49fa-a102-68eaaa979bf8">

2) Add favicon
<img width="61" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/eac63d33-8ba6-4387-b928-d2d4a74d320c">

3) Use .svg for the better qualities of the photos and icons
<img width="240" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/ba436c5d-4499-4243-9444-21b4b5ad6b1f">

4) Add animations for this block
<img width="122" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/06db8a88-59b6-4d27-b3bd-c9c5fd1deb15">

5) For this elements you have animation only when you hovern on them, but when user hovers off them, there is no animation
<img width="176" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/7f610a31-5794-42b5-99ec-2253884b4ef7">

6) You still have scroll in the menue
<img width="1109" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/7f6d4788-ab28-478d-a6b1-628fde99fe12">

7) This block has wrong layout and no hover effects on the elements
<img width="1252" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/9c629373-dd62-48f0-94fa-e4c5e499f0b6">

8) In this block elements also should have hover effect
<img width="1241" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/b023dfe8-fe1f-4b46-ab00-6fa3c0113765">

9) This button should have hover effect
<img width="383" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/4cea0630-f487-4a93-8db9-6336602d9339">

10) This elements should have hover effect
<img width="261" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/80d903c2-8e49-4314-937d-dfa8ac35a60f">

11) When I press on one of the elements I should get or special window to call from the phone, or to send an email, or **separate** page to see the location
<img width="286" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/cc3c4380-1110-4120-b32d-8fa8327414ed">

12) I should be able to send a form

13) Remove auto fill styles for the name and email. They occure when the googl helps you and enteres your name or email for you

14)You have broken layout here on the phone mode
<img width="370" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/136213da-2b5b-4c46-949e-ed120e7053dc">

Fugas252mm commented:
4. incorrect comment, there is an animation when you hover over the icon. 
![Снимок экрана 2023-12-26 205344](https://github.com/mate-academy/layout_landing-page/assets/143828583/7497c9a7-389e-4feb-a4fa-1dfbb8c22f2b)

Fugas252mm commented:
5. Not a correct comment, the animation when you hover over one of the navigation elements is the same as on the Figma layout.
![Снимок экрана 2023-12-26 205951](https://github.com/mate-academy/layout_landing-page/assets/143828583/ddec0922-e6c5-4c09-8950-8720159079fc)
![Снимок экрана 2023-12-26 205955](https://github.com/mate-academy/layout_landing-page/assets/143828583/75c9002d-d449-42c6-abbe-0490f3bed849)

Fugas252mm commented:
11. I have no idea how to do this, I don't know how to use javascript.

Fugas252mm commented:
12. Did everything as in the video from the 'Form' module. 
wrapped all the inputs with the form tag. Added the action attribute - which contains the address where the data from the form will be sent. 
Added the method attribute with the value post - the data is sent in the body of the request.
And I use the name attribute for verification. What did I do wrong?
![Снимок экрана 2023-12-26 215020](https://github.com/mate-academy/layout_landing-page/assets/143828583/0679e7f5-ca87-41fd-9d44-882774b68514)

etojeDenys commented:
> [DEMO LINK](https://Fugas252mm.github.io/layout_landing-page/)

etojeDenys reviewed (changes requested):
1. 
<img width="700" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/d6d11eeb-f377-4045-8cb6-af065118495b">

2. 
<img width="566" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/90017ea8-c39c-425b-b0bc-0f60e23a3c55">

3. 
<img width="754" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/fae2fd43-bef7-4af9-addb-d91e5afc7e02">


4.
<img width="955" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/2163ed47-c044-477f-8221-4375e3857132">

5. 
<img width="662" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/876994f2-c77a-4bd0-a3f4-bf77c62816a3">


6. add some hovers to all links
7. looks different
<img width="1262" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/77dea84c-3310-4cac-8d3d-66c53d87df6b">
<img width="849" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/562f4842-2a8f-4f55-9a42-fc8a1f2e657f">

etojeDenys reviewed (changes requested):
looks good, only few comments:

1. disable textarea resizing 
<img width="1204" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/2ceda77f-5cab-46af-878b-7de0d2f1820d">

2. add some hover to images
3. disable the scroll when menu is open
4. 
<img width="723" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/acb3ef3b-1137-4467-994c-485f2cb4af0a">

anastasiiavorobiova reviewed (changes requested):
Great work! let's make it perfect: 
<img width="1605" alt="Screenshot 2023-12-28 at 16 05 31" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/edaa1b23-5dd7-4cb2-8e0d-eb175c58620c">
<img width="1075" alt="Screenshot 2023-12-28 at 16 05 36" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/20fe79aa-9d04-4c1e-b9c7-4afcf6738cb7">
Consider fixing validation errors 
<img width="714" alt="Screenshot 2023-12-28 at 16 04 39" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/30916bf2-d931-46cd-9f1a-6b448663b27e">
Check the background position for tablets 
<img width="323" alt="Screenshot 2023-12-28 at 16 03 23" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/afa1ffa8-fe75-47c1-8ba6-5613039049bf">
Check margins here. Also, consider adding a proper href attribute for the email link. 
<img width="679" alt="Screenshot 2023-12-28 at 16 03 19" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/6b7aacb4-caed-4186-a71d-888f26466458">
Consider removing default borders here and adding required attributes here 
<img width="200" alt="Screenshot 2023-12-28 at 16 03 10" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/a93b5cd3-c46d-46ce-900b-72254b9f9a52">
Consider exporting an image without a label here 
<img width="451" alt="Screenshot 2023-12-28 at 16 03 00" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/b323750c-5003-4836-bb8f-69a91e705641">
The whole page content moves on hover, consider fixing it 
<img width="81" alt="Screenshot 2023-12-28 at 16 02 08" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/0183f835-8560-4160-b52b-03ac79847799">
Could you please add a hover style here 
<img width="248" alt="Screenshot 2023-12-28 at 16 02 00" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/4587e0f0-7988-4c5a-8262-d1f3304be611">
Check styles here

anastasiiavorobiova reviewed (changes requested):
Please, look carefully through the previous review and try to fix comments here. Feel free to ask for help in the chat or on the QnA session

Fugas252mm commented:
1.Fixed all the errors shown in the screenshots. 
![Снимок экрана 2023-12-28 161609](https://github.com/mate-academy/layout_landing-page/assets/143828583/476a58e0-a16f-4c77-8f9a-8d374f5e1b56)
 2.Corrected the header background image. Now it is centred in the tablet format.
![image](https://github.com/mate-academy/layout_landing-page/assets/143828583/b37ce053-5313-44ff-ba31-5731347b8ac4)
3.Added the right indents and changed the href of the link (href="mailto:example@email.com") to allow sending mail.
![Снимок экрана 2023-12-28 162348](https://github.com/mate-academy/layout_landing-page/assets/143828583/8a2016ef-ff66-41c1-a169-b5fef8278707)
![Снимок экрана 2023-12-28 162359](https://github.com/mate-academy/layout_landing-page/assets/143828583/8c363ca0-92c9-4416-9b8f-1b465e20caa6)
4.Use a clean image.
![Снимок экрана 2023-12-28 162512](https://github.com/mate-academy/layout_landing-page/assets/143828583/a3b92d25-4689-41fc-a35e-1773d274263f)
5.The text stays in place when you hover over the image.
![image](https://github.com/mate-academy/layout_landing-page/assets/143828583/4d5dfe30-3669-430d-8437-137639c97447)
6.Added hover to the side menu icon when hovering.
![image](https://github.com/mate-academy/layout_landing-page/assets/143828583/46e16b28-c0cb-4a39-9d27-507e566e6fae)
7.Checked and changed the animation when you hover over the phone icon.
![image](https://github.com/mate-academy/layout_landing-page/assets/143828583/0c69b66d-4115-48a4-93cd-7e2a55e21996)

Esceype reviewed (changes requested):
GJ!
Still need to fix:
- all fields in form should be required;
- hover effects should affect only the element that is hovered over, but with some of your hover effects it works for the whole page is enlarged or slightly shifted

loralevitska reviewed (approved):
Well done) 
Don't show this number on mobile mode 
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/b85ce466-1f8f-4951-8738-2813ea60e7c2)
